src/ResourceManager/Compute/Commands.Compute/Generated/ComputeAutomationBaseCmdlet.cs

```diff
@@ -36,7 +36,6 @@ public abstract class ComputeAutomationBaseCmdlet : Microsoft.Azure.Commands.Com
         public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
-            ComputeAutomationAutoMapperProfile.Initialize();
```

Comment:
Similar comment.

BTW, @hyonholee This needs update on the code-generator side.

Reply:
@huangpf I will update our code-gen after this PR is merged